experimenter/experimenter/experiments/models.py

```diff
@@ -820,7 +821,58 @@ def clone(self, name, user, rollout_branch_slug=None):
         return cloned
 
     def get_changelogs(self):
-        return self.changes.all()
+        changes_by_date = defaultdict(list)
+        date_option = "%I:%M:%S %p"
+        changelogs = self.changes.order_by("-changed_on").prefetch_related("changed_by")
+        size = len(changelogs) - 1
+
+        for index, changelog in enumerate(changelogs):
+            if index < size:
```

Comment:
You can use slice notation to omit the last changelog, and then you don't need `size`:

```python
for index, changelog in enumerate(changelogs[:-1]):
```

Reply:
The negative indexing isn't supported in `queryset` so had to put it in a list before slicing, please let me know If i missed anything